pages/common/xplr.md

```diff
@@ -10,3 +10,11 @@
 - Focus on a file:
 
 `xplr {{path/to/file}}`
+
+- Focus on a directory:
+
+`xplr --force-focus {{path/to/directory}}`
+
+- Open a directory with selected paths:
```

Comment:
I think we should make clear, that it can't be any path. (It's obvious, but just to make sure)
```suggestion
- Open a directory with selected paths inside of it:
```

Reply:
Hi. It can be any path.

Reply:
Actually need to pass absolute paths, else xple won't find it

Reply:
*absolute or relative to the passed directory.

Reply:
Update: nope. It's absolute paths or any paths relative to the present directory (not the path/to/directory). So, I guess we can keep the example same.

Reply:
The `dolphin` page has this example description: `Launch Dolphin with a specific file or directory selected`.
```suggestion
- Open a directory with specific files or directories selected:
```